nx-libs.spec

```diff
@@ -50,7 +50,7 @@ BuildRequires:  pixman-devel >= 0.13.2
 BuildRequires:  xorg-x11-libX11-devel
 BuildRequires:  xorg-x11-libXext-devel
 BuildRequires:  xorg-x11-libXpm-devel
-BuildRequires:  xorg-x11-libXfont-devel
+BuildRequires:  xorg-x11-libXfont-devel >= 1.4.2
 BuildRequires:  xorg-x11-libXdmcp-devel
 BuildRequires:  xorg-x11-libXdamage-devel
 BuildRequires:  xorg-x11-libXcomposite-devel
```

Comment:
In the general SUSE section, i.e., either after the next `%endif` or before `%if 0%{?suse_version} >= 1130` above, add `BuildRequires: filesystem`. (This is what provides `/usr/share/fonts`.)